### PR TITLE
Ability to set some configurations from .env file

### PR DESCRIPTION
**Pull request by mohamedsabil83**

### CODE REVIEW

File: config/public.php [hunk 1]
```diff
@@ -32,15 +32,15 @@
      * By default, `false`.
      */
 
-    'inline' => false,
+    'inline' => env('LANG_PUBLISHER_INLINE', false),
```
Comment (andrey-helldar) [inline config/public.php:35]:
```suggestion
    'inline' => (bool) env('LANG_PUBLISHER_INLINE', false),
```

File: config/public.php [hunk 2]
```diff
@@ -32,15 +32,15 @@
...
 
     /*
      * Do arrays need to be aligned by keys before processing arrays?
      *
      * By default, true
      */
 
-    'align' => true,
+    'align' => env('LANG_PUBLISHER_ALIGN', true),
```
Comment (andrey-helldar) [inline config/public.php:43]:
```suggestion
    'align' => (bool) env('LANG_PUBLISHER_ALIGN', true),
```

**Review by andrey-helldar — APPROVED** [review]
As a "fool-proof" I added an explicit conversion to a boolean type since the `env` function doesn't always do the job, which has been tested in practice.

In principle, I agree with these changes.

**Comment by andrey-helldar** [comment]
شكراً لك

**Comment by mohamedsabil83** [comment]
Пожалуйста

**Comment by mohamedsabil83** [comment]
> As a "fool-proof" I added an explicit conversion to a boolean type since the `env` function doesn't always do the job, which has been tested in practice.
> 
> In principle, I agree with these changes.

Agreed. Good addition.

**Pull request merged**